src/bin/ion/commands/generate/utils.rs

```diff
@@ -272,6 +309,17 @@ impl Language for RustLanguage {
         namespace.push(type_name.to_case(Case::Snake)); // Add this type's module name to the namespace path
         namespace.push(type_name.to_case(Case::UpperCamel)) // Add this type itself to the namespace path
     }
+
+    fn target_type_as_optional(
+        target_type: FullyQualifiedTypeReference,
+    ) -> FullyQualifiedTypeReference {
+        // TODO: un-comment following block for optional support in Rust, once the templates are changes accordingly
+        // FullyQualifiedTypeReference {
+        //     type_name: vec!["Option".to_string()],
+        //     parameters: vec![target_type],
+        // }
```

Comment:
How is `Option` for optional fields going to interact with nullable ISL types?

Reply:
Currently Rust doesn't support optional field as we completed the Java implementation first.